src/Business Foundation/App/NoSeries/src/Setup/NoSeriesSetupImpl.Codeunit.al

```diff
@@ -126,12 +130,233 @@ codeunit 305 "No. Series - Setup Impl."
             exit(false);
 
         if NoSeriesLine."Increment-by No." <> 1 then begin
-            NextNo := NoSeriesStatelessImpl.IncrementNoText(LastNoUsed, NoSeriesLine."Increment-by No.");
+            NextNo := IncrementNoText(LastNoUsed, NoSeriesLine."Increment-by No.");
             if NextNo > NoSeriesLine."Ending No." then
                 exit(false);
             if StrLen(NextNo) > StrLen(NoSeriesLine."Ending No.") then
                 exit(false);
         end;
         exit(true);
     end;
+
+    procedure ValidateDefaultNos(var NoSeries: Record "No. Series"; xRecNoSeries: Record "No. Series")
+#if not CLEAN24
+#pragma warning disable AL0432
+    var
+        IsHandled: Boolean;
+    begin
+        IsHandled := false;
+        NoSeries.OnBeforeValidateDefaultNos(NoSeries, IsHandled);
+        if not IsHandled then
+#pragma warning restore AL0432
+#else
+    begin
+#endif
+            if (NoSeries."Default Nos." = false) and (xRecNoSeries."Default Nos." <> NoSeries."Default Nos.") and (NoSeries."Manual Nos." = false) then
```

Comment:
```suggestion
            if (not NoSeries."Default Nos.") and (xRecNoSeries."Default Nos." <> NoSeries."Default Nos.") and (not NoSeries."Manual Nos.") then
```
Same for line 172.

Reply:
I agree with your comment but this code is obsolete now and I want to keep it as close to unchanged as I can to avoid accidental breaks.